implementations/rust/ockam/ockam_vault_core/src/types.rs

```diff
@@ -110,6 +116,16 @@ pub struct SecretAttributes {
     length: usize,
 }
 
+impl Default for SecretAttributes {
```

Comment:
I'm not sure we should ever let somebody create such structs without explicitly choosing all parameters

Reply:
Fair enough, yeah. Will remove Default impl